src/test/java/com/revature/services/JMeterServiceTest.java

```diff
@@ -20,23 +19,25 @@
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
 
+import com.revature.docutest.DocutestApplication;
 import com.revature.docutest.TestUtil;
+import com.revature.models.SwaggerDocutest;
 import com.revature.templates.LoadTestConfig;
 
-import io.swagger.models.HttpMethod;
-import io.swagger.models.Operation;
-import io.swagger.models.Path;
-import io.swagger.models.Swagger;
-
 class JMeterServiceTest {
 
     private JMeterService jm;
     private LoadTestConfig loadConfig = new LoadTestConfig();
     private static final String JMeterPropPath = "src/test/resources/test.properties";
     private static final String CSV_FILE_PATH = "./datafiles/user_0.csv";
-    public static final String DIRECTORY_PATH = "./datafiles";
+    private static final String DIRECTORY_PATH = "./datafiles";
+    private static OASService adapter = new OASService();
```

Comment:
So far we still are unable to use `@Autowired` annotations within tests?

Reply:
SpringBootTest runs a lot slower since it needs to set up all of our beans and dependencies. Since these are unit tests, I just set up the service as a static field. I was planning on having a separate class for integration tests using the webapp context.

Reply:
Alrighty, that sounds good.